resume-buddy/src/main/webapp/scripts/resume-review-script.js

```diff
@@ -1,16 +1,19 @@
 /** Run functions when page is loaded */
 function onLoad() {
-  getComments();
+  document.getElementById('comments').style.display = "none";
+  fetch('/user-data')
+    .then(response => response.json())
+    .then(user => {
+      if (((user.matchID).equals("")) == false) { getComments(); }
+    });
 }
 
 /**
  * Fetches comments from the servers and adds them to the DOM.
  */
 function getComments() {
-  /* TODO: default hide comments section
-  /* Only if the user has a match, display comments functionality
-   * https://github.com/googleinterns/ResumeBuddy/issues/67
-   */
+  document.getElementById('match-id').style.display = "none";
+  document.getElementById('comments').style.display = "block";
```

Comment:
I think it would be a good idea to add `display: none` in the CSS to both `#comments` and `#match-info`, and then display the one we want dynamically when the time comes. 

The javascript is changing the `#comments` to disappear right away anyway. And it may be confusing to display `"You do not have a match yet."` while we wait for the fetch to complete. It may be good to have another `<p>` that displays something like `"Loading..."` while we wait instead.

In this version, you would remove line 15 and add `document.getElementById('match-id').style.display = "block";` to an `else` branch around line 7.

Reply:
Or alternatively, keep `#match-id` visibility as it is now, but change it's `innerText` from e.g. "Loading" to "You do not have a match yet."